### ISSUE TITLE
Pulseaudio getting killed when noisetorch starts

### ISSUE THREAD

**Issue by MSVO** [lawl/NoiseTorch#64]
In my system, pulseaudio gets killed when noisetorch runs for a few hours. In order to debug the issue, I started pulseaudio with logging as follows
```
LANG=C pulseaudio -vvvv --log-time=1 > ~/tmp/pulseverbose_1.log 2>&1
```

Then, I started noisetorch from dmenu. It opens the noisetorch ui. I selected the device `Device Name:Built-in Audio Analog Stereo (Device ID: alsa_input.pci-0000_00_1b.0.analog-stereo)` and clicked on `load noisetorch`. At this moment, pulseaudio gets killed.

System info:
```
OS: Manjaro Linux x86_6
Host: SVE15117FNW C903U23A
Kernel: 5.8.11-1-MANJARO
Window Manager: i3
CPU: Intel i5-2450M (4) @ 3.100GHz
GPU: AMD ATI Radeon HD 7550M/7570M/7650M
```

Pulseaudio version
```
pulseaudio 13.0
```

Noisetorch version
```
0.7.3-beta
```

Pulseaudio log:
[pulseverbose_1.log](https://github.com/lawl/NoiseTorch/files/5508195/pulseverbose_1.log)

**Comment by lawl** [comment]
Can you also attach the corresponding noisetorch log file? `/tmp/noisetorch_<timestamp>.log`?

Additionally when you start pulseaudio from a terminal (**not** in daemon mode) and it gets killed, does a line get printed to your terminal saying "Killed"?

**Comment by MSVO** [comment]
Yes, the terminal prints `Killed` for pulseaudio.

Upon starting noisetorch via command line for logging as follows:
```
noisetorch > ~/tmp/noisetorch.log 2>&1
```
It asks for superuser password while loading. It says authentication failed even if I give correct password.
```
==== AUTHENTICATING FOR org.freedesktop.policykit.exec ====
Authentication is needed to run `/home/salih/.local/bin/noisetorch' as the super user
Authenticating as: Muhammed Salih (salih)
Password: 
==== AUTHENTICATION FAILED ====
```
Noisetorch log:
[noisetorch.log](https://github.com/lawl/NoiseTorch/files/5508242/noisetorch.log)

Noisetorch can be loaded successfuly if I start pulseaudio as `pulseaudio --start` and start noisetorch from gui. But after a few hours or so, pulseaudio will get killed. Sometime it gets killed in half an hour. But if I start pulseaudio by logging to file, it gets killed before noisetorch can be loaded.

**Comment by lawl** [comment]
> It asks for superuser password while loading. It says authentication failed even if I give correct password.

This is the issue. Without superuser noisetorch can't adjust the RLIMIT, which can lead to pulseaudio being killed. Sounds like your polkit is misconfigured if it doesn't accept the correct password. This prompt is from `pkexec` and not part of noisetorch.

Maybe [this comment](https://github.com/lawl/NoiseTorch/issues/11#issuecomment-660157059) helps?

**Comment by MSVO** [comment]
Thank you. The problem was with pkexec. I tried adding that rule in given in the comment, somehow it didn't work. Instead, I have added a pkexec noisetorch policy as described in [Noisetorch Wiki: I don't want to enter my password everytime](https://github.com/lawl/NoiseTorch/wiki/I-don't-want-to-enter-my-password-everytime) and it worked. Now noisetorch does not ask for password. I have also made a systemd service for noisetorch by refering to [Noisetorch wiki: Start automatically with Systemd](https://github.com/lawl/NoiseTorch/wiki/Start-automatically-with-Systemd)
